src/components/contacts/contacts.js

```diff
@@ -61,7 +67,13 @@ const Contacts = () => {
         <div>
           <button onClick={() => setModalIsActive(true)}>Add new contact</button>
         </div>
-        <ContactsList contacts={contacts} />
+        <ul className="contact-list">
+          {contacts.map((item, index) => (
+            <li key={index}>
```

Comment:
Don't use the index as a key. https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/
Use id instead